recipe/conda_build_config.yaml

```
@@ -6,3 +6,6 @@ numpy:
   - 1.16
 MACOSX_SDK_VERSION:            # [osx and x86_64]
   - 11.0                       # [osx and x86_64]
+build_type:
+  - release
+  - debug
```

> @mbargull Can you point me in the right direction on how to push to a separate label? I guess I'll need to pass on the `--label` argument to `conda mambabuild` but I can't edit `build_steps.sh` since that's overwritten by `conda-smithy`.

You can add
```suggestion
build_type:
  - release
  - debug
+channel_targets:
+  - conda-forge main
+  - conda-forge debug
zip_keys:
  -
    - build_type
    - channel_targets
```
and rerendering should take care of everything ;).

---

Why do we use a nested list for zip keys? Can't I just do the following?
```yaml
zip_keys:
  - build_type
  - channel_targets
```

---

(Didn't see your comment before.)
The `zip_keys` can have multiple entries (hence a list) and each entry is a list of keys to zip. You can see in the generated `.ci_support/*.yaml` the `c_compiler_version`/`cxx_compiler_version` as another entry being zipped.